### ISSUE TITLE
Karma dependency issue

### ISSUE THREAD

**Issue by pkosenko** [dougjohnston/angular-drum-machine#22]
I installed your app today and NPM module downloads gave me dependency error: "grunt-karma@0.12.1 wants karma@0.13.0 || >= 0.14.0-rc.0".  Your package.json lists : "karma": "~0.12.0".  The issue resolved itself when I ran a grunt-karma upgrade:  "$ npm install grunt-karma@0.13.0"  So you might want to up the package.json karma version to 0.13.0 . . . although i am not yet sure what effect that will have on running grunt-karma (since I haven't done it yet).

**Comment by pkosenko** [comment]
After downloading bower components into the "public/assets" folder (where bower.json is located), I got the following SASS error on running grunt:

$ grunt
Running "sass:dist" (sass) task
DEPRECATION WARNING: Passing --sourcemap without a value is deprecated.
Sourcemaps are now generated by default, so this flag has no effect.
Error: Invalid CSS after "0%": expected keyframes selector, was ":"
        on line 83 of ./public/sass/main.scss
  Use --trace for backtrace.
Warning: Exited with error code 65 Use --force to continue.

But that seems to be the only error (which I will now have to go fix, I guess).  When I used --force, grunt exited the SASS task and went happily through the rest of the tasks and landed in watch mode.
